pootle/apps/pootle_fs/resources.py

```diff
@@ -47,3 +53,96 @@ def unsynced(self):
     @property
     def trackable_stores(self):
         return self.stores.exclude(obsolete=True).filter(fs__isnull=True)
+
+
+class FSProjectStateResources(object):
+
+    def __init__(self, context, pootle_path=None, fs_path=None):
+        self.context = context
```

Comment:
actually it needs `get_fs_path`and a couple of other things so it needs an `FSPlugin` really